src/main/kotlin/stringaddcalculator/domain/StringAddCalculator.kt

```diff
@@ -2,8 +2,9 @@ package stringaddcalculator.domain
 
 object StringAddCalculator {
 
-    fun calculate(tokens: List<String>): Int {
-        return tokens.map(String::toInt)
-            .sum()
+    fun calculate(tokens: List<String>): Number {
+        if (tokens.isEmpty()) return Number(0)
```

Comment:
똑같은 기능의 객체를 매번 생성하기보다는 객체 하나를 재사용하는 것은 어떨까요?

```suggestion
        if (tokens.isEmpty()) return Number.ZERO
```